docs/lineage/lineage-feature-guide.md

```diff
@@ -100,8 +168,8 @@ Using the modal from this view will work the same as described above for editing
 
 ### GraphQL
 
-* [updateLineage](../../graphql/mutations.md#updatelineage)
```

Comment:
why remove this reference to `updateLineage`?

Reply:
+1

Reply:
Good catch - this was a mistake. Adding it back in